### PR TITLE
Add queue labels to separate queue name from label

### PR DESCRIPTION
**Pull request by greg-raymond**

### CODE REVIEW

File: config/prometheus-collectors.php [hunk 1]
```diff
@@ -15,6 +15,10 @@
         'default',
     ],
 
+    'queueLabels' => [
+        'default',
+    ],
```
Comment (jradtilbrook) [inline config/prometheus-collectors.php:21]: I'm thinking we should just merge this array and the one above, with either the keys or the values being the labels to use. What do you think?

File: config/prometheus-collectors.php [hunk 2]
```diff
@@ -12,7 +12,7 @@
     */
 
     'queues' => [
```
Comment (jradtilbrook) [inline config/prometheus-collectors.php:19]: Can you now leave a comment about which way around the key => values are for labelling. Then I think it is ready

**Comment by greg-raymond** [comment]
Just retesting after the changes.

**Review by jradtilbrook — APPROVED** [review]
LGTM

**Comment by greg-raymond** [comment]
You will have to merge it if you approve, as I don't have write access.

**Comment by jradtilbrook** [comment]
Cool. I've just cut release [0.2.0](https://github.com/HealthEngineAU/laravel-prometheus-collectors/releases/tag/0.2.0)

**Pull request merged**